src/app/ui/components/containers/headers/header.tsx

```diff
@@ -8,11 +8,10 @@ import { ArrowLeftIcon, CloseIcon } from '@app/ui/icons';
 import { BigTitleHeader } from './components/big-title-header';
 import { HeaderActionButton } from './components/header-action-button';
 
-type HeaderVariants = 'page' | 'home' | 'onboarding' | 'dialog' | 'bigTitle' | 'fund';
+type HeaderVariants = 'page' | 'home' | 'onboarding' | 'bigTitle' | 'fund';
```

Comment:
I just made a new header for `Dialog` to remove this extra variant here and  `isWaitingOnPerformedAction`